examples/bastion-tunnel-monorepo/.ize/env/testnut/ize.toml

```diff
@@ -1,4 +1,5 @@
 aws_region = "us-east-1"
+aws_profile = "nutcorp-dev"
```

Comment:
This should not be stored in the ize.toml, but rather passed via flags/env vars.

Users can have any profiles.